incubator-tools/line_item_improver/line_items_improver_post_processing.ipynb

```diff
@@ -0,0 +1,918 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #456.                if entity.type == 'line_item':
</pre><p>This should probably be changed to just check if entity contains properties instead of this one specific example</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>